consensus/src/chained_bft/consensus_types/block.rs

```diff
@@ -225,6 +225,14 @@ where
         // A block must carry a QC to its parent.
         checked_precondition_eq!(quorum_cert.certified_block_id(), parent_block.id());
         checked_precondition!(round > parent_block.round());
+
+        // This precondition guards the addition overflow caused by passing
+        // "parent_block.height() + 1" to "new_internal". Since the
+        // "height" and "round" values are always same, the check
+        // "parent_block.height() <= std::u64::MAX - 2" ensures that
+        // the max "height" is consistent with max "round"
+        // (consensus/src/chained_bft/consensus_types/block.rs: "pub fn round").
```

Comment:
no need for the code quote  (and [rustdoc](https://doc.rust-lang.org/rustdoc/what-is-rustdoc.html)) supports imports if you ever need one.